mod_reforged/hooks/skills/actives/barbarian_fury_skill.nut

```diff
@@ -0,0 +1,37 @@
+::mods_hookExactClass("skills/actives/barbarian_fury_skill", function(o) {
```

Comment:
Barbarian Fury only costs 3 AP and 5 Fatigue. I will not fatigue out barbarians in any way at the start of the battle.
Are you you have seen them use it at the very first turn?

Reply:
I don't remember if I have seen Barbarians in particular doing this.